examplefuncsplayer-c/run.sh

```diff
@@ -4,11 +4,15 @@
 
 # we provide this env variable for you
 if [ "$BC_PLATFORM" = 'LINUX' ]; then
-    LIBRARIES="-lutil -ldl -lrt -lpthread -lgcc_s -lc -lm -L../battlecode/c/lib -lbattlecode-linux"
+    LIBRARIES="-lbattlecode-linux -lutil -ldl -lrt -pthread -lgcc_s -lc -lm -L../battlecode/c/lib"
```

Comment:
Do these changes affect correctness or are they just aesthetic?

Reply:
I got some linker errors without those changes: battlecode-linux.so depends on symbols from pthreads and dl, so they must be linked in that order. `-lpthread` -> `-pthread` is mostly aesthetic.